client/src/components/personalInformation.vue

```diff
@@ -42,8 +42,7 @@ export default {
         });
         const getUserInfo = (dataKey: any) => {
             if (props.user) {
-                country.value = props.user.location.country;
-                console.log(country.value);
+                country.value = props.user.location?.country;
```

Comment:
You can load the data from 
```suggestion
                country.value = props.user.location?.name;
```
instead